.env.sample

```diff
@@ -10,11 +10,14 @@ VITE_SDK_VERBOSITY=
 # For usage with contracts version < 5, without the enhanced range for trade by source. In that case - set this to true
 VITE_LEGACY_TRADE_BY_SOURCE_RANGE=
 
+# Opt-in to the gradient branding by setting this flag to true
+VITE_USE_GRADIENT_BRANDING=
```

Comment:
I don't think .env variable is the good place to manage this. Else it'll become the place for every UI stuff. I would recommend using config.ui instead